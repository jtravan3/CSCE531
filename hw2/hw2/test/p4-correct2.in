a=-1+2+(-3)+4+5+6+(-7*8)+(-99/10)+1234
b=a*5/2*3
c=a+b
d=c*c/c
e=d/3*2
f=-(e)
g=-(-(-(-(-(-f)))))
h=-(-(-(-(-(-(-g))))))
