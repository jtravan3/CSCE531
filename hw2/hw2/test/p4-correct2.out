Welcome to the Simple Expression Evaluator.
Enter one expression per line, end with ^D

   1: 1182
   2: 8865
   3: 10047
   4: 10047
   5: 6698
   6: -6698
   7: -6698
   8: 6698
   9: 
